app/lib/ads/ad_info_dialog.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2024 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:sharezone/sharezone_plus/page/sharezone_plus_page.dart';
+import 'package:sharezone_widgets/sharezone_widgets.dart';
+
+void showAdInfoDialog(BuildContext context) async {
+  final navigateToPlusPage = await showDialog<bool>(
+    context: context,
+    builder: (BuildContext context) {
+      return AlertDialog(
+        title: const Text('Werbung in Sharezone'),
+        content: Text.rich(
+          TextSpan(
+            children: [
+              const TextSpan(
+                text:
+                    'Wir führen ein Experiment mit Werbung in Sharezone durch. Wenn du keine Werbung sehen möchten, kannst du ',
```

Comment:
We could commit to a certain time period (e.g. 1-2 months) for the experiment and mention it here so that it might cause less negative reactions (just an idea)

Reply:
Something like:

```suggestion
                    'In den nächsten Wochen führen ein Experiment mit Werbung in Sharezone durch. Wenn du keine Werbung sehen möchten, kannst du ',
```

Or more specific with the timeframe?

Reply:
Yes thats fine, but I'd say: "_Innerhalb_ der nächsten Wochen [...]"